package.json

```diff
@@ -69,11 +69,15 @@
 	"dependencies": {
 		"@babel/runtime": "^7.17.0",
 		"@supabase/supabase-js": "^1.31.1",
-		"dotenv": "^16.0.0",
```

Comment:
Something might have done wrong in syncing with main - this shouldn't remove `dotenv`, it's still used by the backend.

Reply:
thank you, I have added this back in the dependencies.